Source/WebCore/style/StyleTreeResolver.cpp

```diff
@@ -1578,44 +1600,50 @@ std::optional<ResolvedStyle> TreeResolver::tryChoosePositionOption(const Styleab
 
     sortPositionOptionsIfNeeded(options, styleable);
 
-    if (!existingStyle) {
-        options.chosen = true;
-        return ResolvedStyle { RenderStyle::clonePtr(*options.originalStyle) };
```

Comment:
Getting rid of options.originalStyle is nice.

Reply:
Thank you 🙏